README.md

```diff
@@ -43,101 +43,81 @@ While [actors](https://docs.swift.org/swift-book/LanguageGuide/Concurrency.html#
 
 ### Executing asynchronous tasks in FIFO order
 
-Use a `FIFOQueue` to execute asynchronous tasks enqueued from a nonisolated context in FIFO order. Tasks sent to one of these queues are guaranteed to begin _and end_ executing in the order in which they are enqueued.
+Use a `FIFOQueue` to execute asynchronous tasks enqueued from a nonisolated context in FIFO order. Tasks sent to one of these queues are guaranteed to begin _and end_ executing in the order in which they are enqueued. A `FIFOQueue` executes tasks in a similar manner to a `DispatchQueue`: enqueued tasks executes atomically, and a `FIFOQueue` task will deadlock the queue if it awaits results from the queue on which it is executing.
 
-```swift
```

Comment:
This code block didn't add as much as I wanted it to, and with the new API rev on `ActorQueue` it was difficult to showcase the `ActorQueue` API in a similar manner without breaking the `ActorQueue`'s guidelines (i.e. we couldn't show a concise, free-form actor queue without adopting the queue execution context outside of an actor's `init`)